response.go

```diff
@@ -11,10 +11,12 @@ type Response struct {
 	Code      uint32
 	Error     string // error message
 	// Data contains deserialized data for untyped requests
-	Data     []interface{}
-	MetaData []ColumnMetaData
-	SQLInfo  SQLInfo
-	buf      smallBuf
+	Data      []interface{}
+	MetaData  []ColumnMetaData
+	SQLInfo   SQLInfo
+	StmtID    uint64
+	BindCount uint64
```

Comment:
Why do we need these fields in the public API?

Reply:
We don't do if we use only PreparedStatement object's API for all operation with a prepared statement. I will make it lowercase if we reach an agreement to use only PreparedStatement object without any Futures explicitly.